src/client/src/reducers/wizardSelectionReducers/selectWebAppReducer.ts

```diff
@@ -13,8 +13,8 @@ import WizardInfoType from "../../actions/wizardInfoActions/wizardInfoActionType
 const initialState = {
   title: "Fullstack Web Application",
   internalName: "FullStackWebApp",
-  version: "",
-  author: ""
+  version: "v1.0",
+  licenses: ""
```

Comment:
what does this change for?

Reply:
honestly, I'm not 100% sure what these attributes are for, but I made the initial state to match what the state should look like after selecting a full stack web app

Reply:
looking at `generationSelector.ts` we only use the internalName attribute so I dont think version/licenses are even used right now